k8s/netmanager-app/values-prod.yaml

```
@@ -0,0 +1,44 @@
+  pullPolicy: Always
+imagePullSecrets: []
+nameOverride: ''
+fullnameOverride: ''
+service:
+  type: NodePort
+  nodePort: 30113
+  targetPort: 3000
+  port: 3000
+ingress:
+  enabled: false
+nodeSelector: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)